components/channel_members_dropdown/channel_members_dropdown.tsx

```diff
@@ -1,39 +1,63 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import {Channel, ChannelMembership} from 'mattermost-redux/types/channels';
+
+import {UserProfile} from 'mattermost-redux/src/types/users';
+
 import {Constants} from 'utils/constants';
-import * as Utils from 'utils/utils.jsx';
+import * as Utils from 'utils/utils';
 
 import DropdownIcon from 'components/widgets/icons/fa_dropdown_icon';
 import Menu from 'components/widgets/menu/menu';
 import MenuWrapper from 'components/widgets/menu/menu_wrapper';
 
 const ROWS_FROM_BOTTOM_TO_OPEN_UP = 3;
 
-export default class ChannelMembersDropdown extends React.PureComponent {
-    static propTypes = {
-        channel: PropTypes.object.isRequired,
-        user: PropTypes.object.isRequired,
-        currentUserId: PropTypes.string.isRequired,
-        channelMember: PropTypes.object.isRequired,
-        isLicensed: PropTypes.bool.isRequired,
-        canChangeMemberRoles: PropTypes.bool.isRequired,
-        canRemoveMember: PropTypes.bool.isRequired,
-        index: PropTypes.number.isRequired,
-        totalUsers: PropTypes.number.isRequired,
-        actions: PropTypes.shape({
-            getChannelStats: PropTypes.func.isRequired,
-            updateChannelMemberSchemeRoles: PropTypes.func.isRequired,
-            removeChannelMember: PropTypes.func.isRequired,
-            getChannelMember: PropTypes.func.isRequired,
-        }).isRequired,
+export type Props = {
+    channel: Channel;
+    user: UserProfile;
+    currentUserId: string;
+    channelMember: ChannelMembership;
+    isLicensed: boolean;
+    canChangeMemberRoles: boolean;
+    canRemoveMember: boolean;
+    index: number;
+    totalUsers: number;
+    actions: {
+        getChannelStats: (channelId: string) => void;
+        updateChannelMemberSchemeRoles: (channelId: string, userId: string, isSchemeUser: boolean, isSchemeAdmin: boolean) => void;
+        removeChannelMember: (channelId: string, userId: string) => string;
+        getChannelMember: (channelId: string, userId: string) => void;
     };
-
-    constructor(props) {
+}
+export type State = {
+    removing: boolean;
+    serverError: null;
+}
+export default class ChannelMembersDropdown extends React.PureComponent<Props, State> {
+    // static propTypes = {
```

Comment:
We can remove this block entirely, TS removes the need for `PropTypes` (in our case)